world.go

```diff
@@ -0,0 +1,315 @@
+package ibctest
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/cosmos/cosmos-sdk/crypto/hd"
+	"github.com/cosmos/cosmos-sdk/crypto/keyring"
+	"github.com/cosmos/cosmos-sdk/types"
+	"github.com/ory/dockertest/v3"
+	"github.com/strangelove-ventures/ibctest/ibc"
+	"github.com/strangelove-ventures/ibctest/testreporter"
+)
+
+// World represents a full IBC network, encompassing a collection of
+// one or more chains, one or more relayer instances, and initial account configuration.
+type World struct {
```

Comment:
I could be convinced `World` is the right name. 

But wanted to also suggestion `NewGraph`, `NewNetwork`, or perhaps `NewTestSuite`.

Reply:
My vote as proposed by @joeabbey : `NewInterchain`